packages/meditrak-server/src/routes/importSurveys/ConfigImporter/processAutocompleteConfig.js

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
In here there's some quite familiar code - is there stuff we can reuse in a common util from your earlier PRs?